sw/host/opentitantool/src/command/gpio.rs

```diff
@@ -75,11 +75,11 @@ pub struct GpioSetDirection {
     pub pin: String,
     #[structopt(
         name = "DIRECTION",
-        possible_values = &PinDirection::variants(),
+        possible_values = &PinMode::variants(),
         case_insensitive=true,
-        help = "The I/O direction of the pin"
+        help = "The I/O mode of the pin"
     )]
-    pub direction: PinDirection,
+    pub direction: PinMode,
 }
 
 impl CommandDispatch for GpioSetDirection {
```

Comment:
Rename this struct as `GpioSetMode` (and update the spec document).

Reply:
Done